src/integrationTest/sam.test.ts

```diff
@@ -202,7 +202,7 @@ async function getAddConfigCodeLens(documentUri: vscode.Uri): Promise<vscode.Cod
                 return codeLenses || []
             }
         } catch (e) {
-            console.log(`sam.test.ts: getAddConfigCodeLens() on "${documentUri.fsPath}" failed, retrying:\n${e}`)
+            console.log(`sam.test.ts: getAddConfigCodeLens(): failed, retrying:\n${e}`)
```

Comment:
maybe we should keep the extra info?

Reply:
Yeah might as well. The paths were just really long and so not immediately useful (you couldn't see the actual filename without scrolling to the right). This code block also keeps running even if the CodeLens test fails, which is not desirable at all.

Reply:
> This code block also keeps running even if the CodeLens test fails, which is not desirable at all.

Hmm I wonder why we did that...

Reply:
> This code block also keeps running even if the CodeLens test fails, which is not desirable at all.

This code is from before `waitUntil()` was introduced. Maybe we should switch to that.

Reply:
I already have a PR up for some integration test changes so I'll do it in there.